src/registrar/fixtures.py

```diff
@@ -79,6 +79,7 @@ class UserFixture:
             "username": "319c490d-453b-43d9-bc4d-7d6cd8ff6844",
             "first_name": "Rachid-Analyst",
             "last_name": "Mrad-Analyst",
+            "email": "rachid.mrad@gmail.com",
```

Comment:
any reason you just added an email to staff not admin as well? I would think with this setup you could do both in case you log in as a user via your admin level account

Reply:
Just to confirm w/ this change, if we add our own email(s) for Staff (and possibly Analyst?), it will dynamically assist us with emails, correct?

Reply:
Correct @therealslimhsiehdy. No reason @abroddrick other than I've gotten in the habit of testing notifications on my Analyst user, triggered from my Superuser user.